apps/ledger-live-mobile/src/families/ethereum/EditFeeUnitEthereum.js

```diff
@@ -83,7 +83,7 @@ export default function EditFeeUnitEthereum({
           >
             <LText style={[styles.currencyUnitText, { color: colors.live }]}>
               <CurrencyUnitValue
-                unit={feeCustomUnit || mainAccount.unit}
+                unit={mainAccount.unit || feeCustomUnit}
```

Comment:
mmh i'm not sure this is the fix to be done.
feeCustomUnit **used to be** a feature in send flow that was allowing to configure the fee in place. it sounds like to be tech debt now, so you could completely remove the `|| feeCustomUnit` (mainAccount.unit will **always** be set, so the || isn't going to work)

cc @haammar-ledger for scheduling work in this "tech debt" to remove (the feeCustomUnit can be removed i think, if you confirm it's not used in the UI)

Reply:
Task created https://ledgerhq.atlassian.net/browse/LIVE-2747
Thanks for the feedback.